src/analyses/goto_check.cpp

```diff
@@ -1296,10 +1296,29 @@ void goto_checkt::bounds_check(
 
     binary_relation_exprt inequality(effective_offset, ID_lt, size_casted);
 
+    exprt::operandst alloc_disjuncts;
+    for(const auto &a : allocations)
+    {
+      typecast_exprt int_ptr{pointer, a.first.type()};
+
+      binary_relation_exprt lb_check{a.first, ID_le, int_ptr};
```

Comment:
Minor nit: 'lb' and 'ub' are not the most obvious naming abbreviations when reading this code outside of the context of a PR that draws attention to the fact that these are bounds checks :-)

Reply:
Indeed - I'm now spelling out "lower_bound", "upper_bound", etc.